doc/source/changed_in_8.0.rst

```diff
@@ -0,0 +1,171 @@
+
+Features removed from |Percona Server| 8.0
+================================================================================
+
+Some features, that were present in |Percona Server| 5.7, are removed from
+|Percona Server| 8.0:
+
+Removed Features
+--------------------------------------------------------------------------------
+
+- Slow Query Log Rotation and Expiration
+- CSV engine mode for standard-compliant quote and comma parsing
+- Enforcing a storage engine
+- Utility user
+- Expanded program option modifiers
+- The ``ALL_O_DIRECT`` InnoDB flush method: it is not compatible with the new
+  redo logging implementation
+- XTRADB_RSEG table from INFORMATION_SCHEMA
+- InnoDB memory size information from SHOW ENGINE INNODB STATUS; the same
+  information is available from Performance Schema memory summary tables
+
+.. seealso::
+
+   |MySQL| Documentation: Performance Schema Table Description
+      https://dev.mysql.com/doc/refman/8.0/en/performance-schema-table-descriptions.html
+
+.. TODO: Redo logging
+
+The enforcement of the storage engine has been removed not to conflict with the
+storage engine blacklisting feature in |MySQL| 8.0.
+
+Removed Syntax
+--------------------------------------------------------------------------------
+
+- The ``SET STATEMENT ... FOR ...`` statement that enabled setting a variable
+  for a single query.
+- The ``LOCK TABLES FOR BACKUP`` statement due to the introduction of the
+  ``log_status`` table in Performance Schema of |MySQL| 8.0.
+
+.. rubric:: Replacing ``SET STATEMENT FOR`` with the Upstream Equivalent
+
+|Percona Server| 8.0 uses the ``SET_VAR`` as the equvalent introduced in |MySQL|
+8.0. ``SET_VAR`` is an optimizer hint that can be applied to session variables.
+
+|Percona Server| 8.0 extends the ``SET_VAR`` hint to support the following:
+
+- The ``OPTIMIZE TABLE`` statement
+- MyISAM session variables
+- Plugin or Storage Engine variables
+- InnoDB Session variables
+- The ``ALTER TABLE`` statement
+- ``CALL stored_proc()`` statement
+- The ``ANALYZE TABLE`` statement
+- The ``CHECK TABLE`` statement
+- The ``LOAD INDEX`` statement (used for MyISAM)
+- The ``CREATE TABLE`` statement
+
+|Percona Server| 8.0 also supports setting the following variables by using ``SET_VAR``:
+
+- innodb_lock_wait_timeout
+- innodb_tmpdir
+- innodb_ft_user_stopword_table
+- block_encryption_mode
+- histogram_generation_max_mem_size
+- myisam_sort_buffer_size
+- myisam_repair_threads
+- myisam_stats_method
+- preload_buffer_size (used by MyISAM only)
+  
+.. seealso::
+
+   |MySQL| Documentation: The log_status Table
+      https://dev.mysql.com/doc/refman/8.0/en/log-status-table.html
+   |MySQL| Documentation: Variable-setting hint syntax
+      https://dev.mysql.com/doc/refman/8.0/en/optimizer-hints.html#optimizer-hints-set-var
+
+Removed Plugins
+--------------------------------------------------------------------------------
+
+- ``SCALABILITY_METRICS``
+- ``QUERY_RESPONSE_TIME_AUDIT``
```

Comment:
mysql> SHOW PLUGINS;
Don' refer to QRT by this particular name, it leaves out three other plugins from the set:
...
| QUERY_RESPONSE_TIME         | ACTIVE   | INFORMATION SCHEMA | query_response_time.so | GPL     |
| QUERY_RESPONSE_TIME_AUDIT   | ACTIVE   | AUDIT              | query_response_time.so | GPL     |
| QUERY_RESPONSE_TIME_READ    | ACTIVE   | INFORMATION SCHEMA | query_response_time.so | GPL     |
| QUERY_RESPONSE_TIME_WRITE   | ACTIVE   | INFORMATION SCHEMA | query_response_time.so | GPL     |
+-----------------------------+----------+--------------------+------------------------+---------+